CaptureSDK/GiniCaptureSDKPinningExample/Example Swift/AppDelegate.swift

```diff
@@ -27,4 +27,11 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
         coordinator.processExternalDocument(withUrl: url, sourceApplication: options[.sourceApplication] as? String)
         return true
     }
+
+    func application(_ application: UIApplication, supportedInterfaceOrientationsFor window: UIWindow?) -> UIInterfaceOrientationMask {
+        if UIDevice.current.userInterfaceIdiom == .pad {
```

Comment:
Same

Reply:
We should allow portrait orientation only for phones

Reply:
All good, @chriskryn just pick the check from the existing extension 😊